Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2820,6 +2821,53 @@ static inline bool isFlexOrGridItem(RenderObject* renderer)
     return box.isFlexItem() || box.isGridItem();
 }
 
+static Ref<CSSValue> valueForScrollTimelineAxis(const Vector<ScrollAxis>& axes)
+{
+    if (axes.isEmpty())
+        return CSSPrimitiveValue::create(CSSValueNone);
```

Comment:
```suggestion
    if (axes.isEmpty())
        return CSSPrimitiveValue::create(CSSValueBlock);
```

`none` isn't valid for `scroll-timeline-axis`

Reply:
Thanks for catching this!